system-test/test-middleware-express.ts

```diff
@@ -26,23 +27,24 @@ const logging = new Logging();
 
 const WRITE_CONSISTENCY_DELAY_MS = 20 * 1000;
 const TEST_TIMEOUT = WRITE_CONSISTENCY_DELAY_MS + (10 * 1000);
+const LOG_NAME = `bunyan-system-test-${uuid.v4()}`;
 
 describe('express middleware', () => {
   let logger: elb.MiddlewareReturnType['logger'];
   let mw: elb.MiddlewareReturnType['mw'];
 
   before(async () => {
-    ({logger, mw} = await elb.middleware({level: 'info'}));
+    ({logger, mw} = await elb.middleware({logName: LOG_NAME, level: 'info'}));
   });
 
   describe('global logger', () => {
     it('should properly write log entries', async () => {
-      const LOG_MESSAGE = 'test log message';
+      const LOG_MESSAGE = `unique log message ${uuid.v4()}`;
       logger.info(LOG_MESSAGE);
 
       await delay(WRITE_CONSISTENCY_DELAY_MS);
 
-      const log = logging.log('bunyan_log');
+      const log = logging.log(`${LOG_NAME}_applog`);
```

Comment:
Why is this suffixed with `_applog` for middleware? For non middleware the it just uses `${LOG_NAME}`. So now we have two different  naming conventions for the same app's logs.

Reply:
Yes, this is suboptimal. The reason behind this is that for request bundling to work, the 'request log stream' (the log which has the http requests) has to be a different log name than the the 'application log' (where normal user logging should go). We chose to use a suffix for the applog, but it might have been better to use a suffix for the request log (since it is not user-generated).

This will be fixed when a counterpart of [this PR](https://github.com/googleapis/nodejs-logging-winston) is ported over to this library.

Note that you will still have two different log streams.